slide-in-with-cta/template.css

```
@@ -0,0 +1,204 @@
+/* ==========================================================================
+   Slide-In
+========================================================================== */
+
+#evg-slide-in-with-cta {
+    position: absolute;
+    z-index: 100000;
```

I noticed that we haven't been staying consistent with z-index values across templates. Could you please make a separate commit that updates the z-index of the two exit intent popup templates (both use "9999999") and the infobar template (uses "10000000")? Let's use "1000000" for them all.

```suggestion
    z-index: 1000000;
```

---

@nriserEG maybe the infobars should be less than the pop-ups and the slide-in? I kinda feel like those should be above an infobar if they happen to overlap, right?

---

@amaskiellEG Good call 👍 How about 100000 for infobars, 1000000 for slide-in, and 10000000 for popups (which could have a lightbox overlay that we wouldn't want anything overlapping on top of)? I think it'd look better than doing something like 100001, 100002, 100003

---

Sounds good to me :)

---

For reference: https://github.com/evergage/evergage-global-templates/pull/23/commits/7affe582eac2607c2b31e26dd082e189b0e29445